### DISCUSSION TITLE
Add a number input field to spacer@cinnamon.org applet

### DISCUSSION THREAD

**Discussion by igorsantos07** [linuxmint/discussions#643] [Ideas]
It would be easier to have matching spacers, than having to drag the handle time and time again :)

**Comment by anaximeno** [comment]
I didn't completely understand the request, how it should look/work, and what the goal of the number field on the spacer applet would be.

**Comment by JosephMcc** [comment]
Don't really see the point in this. The slider displays the number. Isn't really going to add anything.

**Reply by Secret-chest** [reply]
It doesn't allow you to set it.
